locksmith/src/controllers/v2/metadataController.ts

```diff
@@ -86,7 +84,7 @@ export class MetadataController {
 
   async getKeyMetadata(request: Request, response: Response) {
     try {
-      const { keyId } = request.params
+      const keyId = request.params.keyId.toLowerCase()
```

Comment:
keyId should be a number? Why the lower casing?

Reply:
This is how it is done on the previous endpoint. I wasn't sure if the key were ever going to be non-integer so copied it from there.